samples/tfx/taxi-cab-classification-pipeline.py

```diff
@@ -157,18 +157,18 @@ def taxi_cab_classification(
     analyze_slice_column='trip_start_hour'):
 
   tf_server_name = 'taxi-cab-classification-model-{{workflow.name}}'
-  validation = dataflow_tf_data_validation_op(train, evaluation, column_names, key_columns, project, mode, output)
+  validation = dataflow_tf_data_validation_op(train, evaluation, column_names, key_columns, project, mode, output).apply(gcp.use_gcp_secret('user-gcp-sa'))
   preprocess = dataflow_tf_transform_op(train, evaluation, validation.outputs['schema'],
-      project, mode, preprocess_module, output)
+      project, mode, preprocess_module, output).apply(gcp.use_gcp_secret('user-gcp-sa'))
   training = tf_train_op(preprocess.output, validation.outputs['schema'], learning_rate,
-      hidden_layer_size, steps, 'tips', preprocess_module, output)
+      hidden_layer_size, steps, 'tips', preprocess_module, output).apply(gcp.use_gcp_secret('user-gcp-sa'))
   analysis = dataflow_tf_model_analyze_op(training.output, evaluation,
-      validation.outputs['schema'], project, mode, analyze_slice_column, output)
+      validation.outputs['schema'], project, mode, analyze_slice_column, output).apply(gcp.use_gcp_secret('user-gcp-sa'))
```

Comment:
Would it be better to have a "analysis.apply(gcp.use_gcp_secret('user-gcp-sa'))" in a separate line? I think the line is too long.

Reply:
Done.